pmix_governance.tex

```diff
@@ -425,6 +425,30 @@ \subsubsection{ASC Secretaries}\label{asc-secretaries}}
 Alternate for a Member may vote on behalf of the Member subject to the
 one vote per Member limit.
 
+\hypertarget{asc-guidance}{%
+\subsection{ASC Guidance}\label{asc-guidance}}
+
+This section provides general guidance for the PMIx ASC and participants.
+The rationale behind providing this guidance is to outline
+philosophy and best practices the ASC should follow in 
+maintaining the PMIx Standard. 
+
+\hypertarget{release-schedule-guidance}{%
+\subsubsection{Release Schedule Guidance}\label{release-schedule-guidance}}
+
+As described in Section~\ref{release-managers},
+the schedule of major releases of the PMIx
+Standard is determined by the Release Managers in coordination with other

```

Comment:
Not sure if "Release Mangers" should be capitalized. It's not a proper name.

Reply:
"Release Managers" is capitalized throughout the document to indicate that it is a formal position in the ASC. If we want to change that -- that is a bigger (and IMO separate) change